apps/builder/app/builder/features/style-panel/shared/property-name.tsx

```diff
@@ -276,3 +287,62 @@ export const PropertyName = ({
     </Flex>
   );
 };
+
+type PropertyNameProps = {
+  style: StyleInfo;
+  properties: StyleProperty[];
+  label: string | ReactElement;
+  title?: string;
+  description?: React.ReactNode;
+  onReset: () => void;
+};
+
+export const PropertyName = ({
+  style,
+  title,
+  description,
+  properties,
+  label,
+  onReset,
+}: PropertyNameProps) => (
+  <PropertyNameInternal
+    style={style}
+    title={title}
+    description={description}
+    properties={properties}
+    label={label}
+    onReset={onReset}
+  />
+);
+
+type NonResetablePropertyNameProps = {
+  style: StyleInfo;
+  properties: StyleProperty[];
+  label: string | ReactElement;
+  title?: string;
+  description?: React.ReactNode;
+  disabled?: boolean;
+};
+/**
+ * Some properties like layered background-image, background-size are non resetable.
+ * UI of background would be unreadable, imagine you have
+ * background-size inherited from one source, background-image from the other,
+ * Every property have different amount of layers. The final result on the screen would be a mess.
+ */
+export const NonResetablePropertyName = ({
```

Comment:
Looks like this component doesn't introduce any logic. Maybe just use PropertyName?

Reply:
Created for comment, as otherwise bakground tooltips would have questions